labs/b5.md

```diff
@@ -0,0 +1,364 @@
+---
+title: Lab 5 - Networking 101
+layout: lab
+---
+
+## Overview
+It is undeniable that the internet is an important system that has redefined our world. The ability to develop networks and allow devices to communicate is critical to value of modern day computer systems.  This lab will take a look into the basics of computer networking and then examine networks through the perspective of a sysadmin.
+
+We will be using web browsing as an analogy to understand the basics of networking. What exactly happens when I go web browsing for cat pictures?
+
+But first let's take a short dive into the details of networking.
+
+---
+
+## OSI Model
+To lay the groundwork for understanding networking, we'll first turn to the Open Systems Interconnection model (OSI).  The OSI model describes modern computer systems by partitioning them into several 'layers':
+1. **Physical Layer**
+This layer deals with the physical transmission of the data such as passing electrical signals over a fiber optic cable or radio frequencies for wireless.
+2. **Data Link Layer**
+This layer deals with transfer data between network nodes in a wide area network (WAN) or a local area network (LAN). An example of this is L2 routing according to MAC addresses.
```

Comment:
It might be a little confusing to say L2 is a WAN technology, since the L2 addresses change every hop. I'd say something along the lines of L2 doing local routing inside a LAN and L3 oriented towards global routing across multiple LANs

Reply:
I can see what you're going for but WANs can be purely L2.  This may be more complicated than it needs to be so I will fix wording to be in the context of local and global routing.